shell/platform/fuchsia/runtime/dart/utils/tempfs.cc

```diff
@@ -45,9 +45,10 @@ void RunnerTemp::Start() {
     zx_status_t status = memfs_install_at_with_page_limit(
         loop_->dispatcher(), kMaxTmpPages, kTmpPath);
 #else
+    memfs_filesystem_t* fs;
     // Hot reload uses /tmp to hold the updated dills and assets so do not
     // impose any size limitation in non product runners.
-    zx_status_t status = memfs_install_at(loop_->dispatcher(), kTmpPath);
+    zx_status_t status = memfs_install_at(loop_->dispatcher(), kTmpPath, &fs);
```

Comment:
do we need to dispose of this handle using `memfs_uninstall_unsafe`? I can't find a clear answer reading the headers, but the memfs unit tests seem to: https://cs.corp.google.com/fuchsia/fuchsia-mirror/zircon/system/utest/memfs/memfs-tests.cc?q=memfs_uninstall_unsafe&dr=C&l=179

Reply:
I noticed that too. I've reached out to the author. Will update based on this take.